src/libraries/Common/src/System/Collections/Generic/EnumerableHelpers.Linq.cs

```diff
@@ -11,32 +11,6 @@ namespace System.Collections.Generic
     /// </summary>
     internal static partial class EnumerableHelpers
     {
-        /// <summary>
-        /// Tries to get the count of the enumerable cheaply.
-        /// </summary>
-        /// <typeparam name="T">The element type of the source enumerable.</typeparam>
-        /// <param name="source">The enumerable to count.</param>
-        /// <param name="count">The count of the enumerable, if it could be obtained cheaply.</param>
-        /// <returns><c>true</c> if the enumerable could be counted cheaply; otherwise, <c>false</c>.</returns>
-        internal static bool TryGetCount<T>(IEnumerable<T> source, out int count)
```

Comment:
Could we keep this and call it from the public method after null checks? (since internal calls don't need those)